src/components/Homepage/HomePage.jsx

```diff
@@ -0,0 +1,61 @@
+import React, { useEffect } from "react";
+import { useState } from "react";
+import { Link } from "react-router-dom";
+const axios = require("axios");
+
+const api_key = process.env.REACT_APP_API_KEY;
+
+export default function HomePage({ setMovieInfo }) {
+  const [movies, setMovies] = useState([]);
+
+  useEffect(() => {
+    fetchData();
+  }, []);
+
+  async function fetchMovieData(id) {
+    const response = await axios.get(
```

Comment:
You should handle errors in every network request.

Reply:
fixed